services/cctp-relayer/relayer/relayer.go

```diff
@@ -593,9 +673,10 @@ func (c *CCTPRelayer) submitReceiveCircleToken(parentCtx context.Context, msg *r
 	}
 
 	// Store the completed message.
-	// Note: this can cause double sbumissin sometimes
+	// Note: this can cause double submission sometimes
 	msg.State = relayTypes.Submitted
 	msg.DestNonce = int(nonce)
+	msg.DestTxHash = destTxHash.String()
```

Comment:
This won't work accurately because of the way submitter works rn, but we can deal w/ this later

Reply:
This should be ready w/ the merging of #1098